scripts/zfs-tests.sh

```diff
@@ -695,17 +695,21 @@ fi
 #
 # Run all the tests as specified.
 #
+EVILRES=$(mktemp -u -t test-runner-error)
```

Comment:
This is both (notionally) unsafe and doesn't work with coreutils mktemp:
```
mktemp: too few X's in template ‘test-runner-error’
```

Suggest
```sh
EVILRES=$(mktemp -t test-runner-error.XXXXXX)
```
(which does create /tmp/t-r-e.XXXXXX.abcdef on FreeBSD (but who cares) and Just Works on coreutils)